pubspec.yaml

```diff
@@ -63,8 +63,12 @@ dependencies:
   firebase_core: ^2.14.0
 
 dev_dependencies:
+  flutter_driver:
+    sdk: flutter
```

Comment:
deps: Add integration_test and flutter_driver, in preparation for integration tests

Following the style of past commits in this section, it's helpful to flag that these are only dev dependencies (and not dependencies that go into the published app):

> deps: Add dev:integration_test, dev:flutter_driver

Cf the other previous commits that added entries here:
381c80230 deps: Add dev:fake_async
483a3ef1d deps: Add drift, sqlite3_flutter_libs, path_provider, path, dev:drift_dev
7a9bf81c5 deps: Add dev:checks
6c9b94c45 deps: Add dev:test
9ea09ceb6 deps: Add json_annotation, dev:json_serializable, dev:build_runner

I found those with `git log -L 65,85:pubspec.yaml`. (Where the line range corresponds to the `dev_dependencies:` section. There are fancier ways to write `git log -L` ranges in terms of regexps, which could make the command robust to different versions of the file; but in ad hoc interactive use, I'd just read off the line numbers from looking at the file in my editor.)

The explanation about why we're adding these packages can then go in the body of the commit message. Though in this instance I'd be happy to leave it out — one of the package names already tells the reader it's about integration tests.

Reply:
BTW the `dev:` notation there is from the syntax accepted by `flutter pub add` (or `dart pub add`). See `flutter pub add --help`.